src/model-summary/index.jsx

```diff
@@ -20,7 +23,9 @@ function ModelProperty({ property, model, formatters, specification, formatNullV
 
     return (
         <Fragment>
-            <dt><Snippet {...snippetProps}>{`fields.${property}.label`}</Snippet></dt>
+            {property === APPLICANT_TRAINING_USE_AT_WORK || property === APPLICANT_LEARNING_USE ?
+                <dt><Snippet {...snippetProps}>{`fields.${property}.checkAnswerLabel`}</Snippet></dt> :
+                <dt><Snippet {...snippetProps}>{`fields.${property}.label`}</Snippet></dt>}
             <dd>
```

Comment:
This should not have overrides for specific fields as that is hard to follow. Instead checkAnswerLabel should be a generic property. The Snippet compoent has a fallback for this use case ([snippet/index.jsx](https://github.com/UKHomeOffice/asl-components/blob/main/src/snippet/index.jsx))

```suggestion
                <dt>
                  <Snippet {...snippetProps} fallback={`fields.${property}.label`}>
                    {`fields.${property}.checkAnswerLabel`}
                  </Snippet>
                </dt> :
            <dd>
```

Reply:
that doesn't work. I only want different label in 2 places and in every other place I want other label. I will call you to discuss!

Reply:
thanks for your help @jeff-horton-ho-sas .